docs/development/core/public/kibana-plugin-public.uisettingsstart.md

```diff
@@ -8,5 +8,5 @@
 <b>Signature:</b>
 
 ```typescript
-export declare type UiSettingsStart = UiSettingsClient;
+export declare type UiSettingsStart = PublicMethodsOf<UiSettingsClient>;

```

Comment:
hm. not really useful typing without a link. could `{@link UiSettingsClient}` help in this case? @joshdover  I remember you added links the a couple days ago

Reply:
Good point. Looking at the docs for UiSettings I also felt that having two types which describe exactly the same thing is confusing, if you know how the one works you know how the other works, so better to have one name so that it's obvious that it's the same thing. So I change these types into one type called `UiSettingsClientContract`